scanners/zap-automation-framework/examples/demo-zap-baseline-migrate-scan/scan.yaml

```
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: zap-baseline-automation-framework-config
+data:
+  automation.yaml: |-
+
+    env:                                   # The environment, mandatory
+      contexts :                           # List of 1 or more contexts, mandatory
+        - name: zap-baseline-automation-scan # Name to be used to refer to this context in other jobs, mandatory
+          urls: ["http://juiceshop.demo-targets.svc:3000/"] # A mandatory list of top level urls, everything under each url will be included
+    jobs:
+      - type: spider                       # The traditional spider - fast but doesnt handle modern apps so well
+        parameters:
+          context: zap-baseline-automation-scan # String: Name of the context to spider, default: first context
+          maxDuration: 1                   # Int: The max time in minutes the spider will be allowed to run for, default: 0 unlimited
+      - type: passiveScan-wait             # Passive scan wait for the passive scanner to finish
+        parameters:
+          maxDuration: 5                   # Int: The max time to wait for the passive scanner, default: 0 unlimited
+      - type: report                       # Report generation
+        parameters:
+          template: traditional-xml                        # String: The template id, default : modern
+          reportDir: /home/securecodebox/               # String: The directory into which the report will be written
+          reportFile: zap-results                     # String: The report file name pattern, default: {{yyyy-MM-dd}}-ZAP-Report-[[site]]
+        risks:                             # List: The risks to include in this report, default all
+          - high
+          - medium
+          - low
+
+---
+apiVersion: "execution.securecodebox.io/v1"
+kind: Scan
+metadata:
+  name: "zap-automation-framework-juice-shop"
+  labels:
+    organization: "OWASP"
+spec:
+  scanType: "zap-automation-framework"
+  parameters:
+    - "-autorun"
+    - "/home/securecodebox/scb-automation/1-automation.yaml"
+  volumeMounts:
+    - name: zap-baseline-automation-framework-config
+      mountPath: /home/securecodebox/scb-automation/1-automation.yaml
+      subPath: 1-automation.yaml
```

the 1- naming doesn't really make sense here anymore as the automation framework doesn't have the same overwriting mechanism

```suggestion
      subPath: automation.yaml
```